netlify.toml

```diff
@@ -0,0 +1,28 @@
+[Settings]
+  ID = "tilejson"
+
+[build]
+  base    = "./src/app"
+  publish = "./src/dist"
+  command = "pushd ../.. && ./scripts/cibuild && popd"
+  environment = { CI = "1", YARN_VERSION = "1.12.3" }
+  
+# [[redirects]]
```

Comment:
Should this commented out code be deleted?

Reply:
This wasn't part of my PR. I think ops might need it to completely move to Netlify so I'm going to leave it in for now.